### DISCUSSION TITLE
Winpilot (BloatynosyAI) 3.0.5

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#150] [Announcements]
### Important note: Bloatynosy will soon become Winpilot. Check out [3.0.0](https://github.com/builtbybel/Winpilot/releases/tag/3.0.0) for more details 

- The CoTweaker plugin is leveling up and bringing back almost all the old Bloatynosy privacy and system settings in a sleek, optimized form.

![Screenshot 2024-03-17 211634](https://github.com/builtbybel/Winpilot/assets/57478606/6e4137e4-f089-45f3-b183-c5cc20d49420)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Winpilot/releases/tag/3.0.5'>Winpilot (BloatynosyAI) 3.0.5</a>.</em>

**Comment by TriflingToad** [comment]
Why so many name changes? I've followed this program from whatever was before ThisIsWin11.

**Comment by TriflingToad** [comment]
also it's pretty overwhelming compared to the old GUI, I ended up just downloading an old version to delete edge
